oada/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "strict": true,
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "lib": ["ES2019"],
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true
+  }
+}
```

Comment:
I am all for converting more services to TS, but we should have a share tsconfig they extend. Preferably is would be based off the `@tsconfig` for which node we want to target.

Reply:
Fully agree. This was more of a "play with nats" experiment and I wanted to play in TS, so I just throw this here. The more we can take from @tsconfig the better.

Reply:
I'd say we use `@tsconfig/node12` and then turn on all the noImplict rules. Technically node10 is still in LTS for a couple more months but I say we ditch it now.

We could just go with node 14 since that is in our images, but maybe people will want to run it outside docker.